specs/front_desk_spec.rb

```diff
@@ -0,0 +1,99 @@
+require_relative 'spec_helper'
+
+describe "FrontDesk class" do
+
+  describe "initialize" do
+
+    it "is an instance of FrontDesk" do
+      admin = Hotel::FrontDesk.new
+      admin.must_be_kind_of Hotel::FrontDesk
+    end
+
+    it "loads a master list of all rooms" do
+      admin = Hotel::FrontDesk.new
+
+      admin.rooms.length.must_equal 20
+      admin.rooms[0].number.must_equal 1
+    end
+
+  end
+
+  describe "make_reservation method" do
+
+    before do
+      @front_desk_1 = Hotel::FrontDesk.new
+    end
+
+    it "stores new reservation" do
+      dates = { start_date: Date.new(2018,6,7), end_date: Date.new(2018,6,10)}
+
+      new_res = @front_desk_1.make_reservation(dates)
+
+      @front_desk_1.find_reservations_for( Date.new(2018, 6, 8))
+        .must_include new_res
+
+      new_res.start_date.must_equal Date.new(2018, 6, 7)
+    end
+
+
+    it "adds the reservation to front desk's collection of reservations"
```

Comment:
Not sure if you really need this test, you already test whether it was successfully reserved above.